project.xml

```diff
@@ -13,10 +13,10 @@
     <use project = "libzmq" />
     <use project = "uuid" optional = "1" implied = "1" />
     <use project = "systemd" optional = "1" implied = "1" min_major = "200" />
-    <target name = "*" />
```

Comment:
Yes, my zproject rpm was broken ... I did drop it in order to continue with a development. Will bring it back.